source/security-and-authentication/mongodb-aws-auth.txt

```diff
@@ -24,8 +31,8 @@ AWS IAM credentials, see the guide on :atlas:`How to Set Up Unified AWS Access <
 
 .. important::
 
-   You must use {+connector+} version 1.5 of later to connect to a MongoDB
-   server set up to authenticate using your AWS IAM credentials. AWS IAM
+   You must use {+connector+} version 1.5 of later to connect to a {+cluster+}
+   set up to authenticate using your AWS IAM credentials. AWS IAM
    credential authentication is available in MongoDB server version 4.4
    and later.
```

Comment:
I left "MongoDB server" here because I think it makes sense with a version number.